### ISSUE TITLE
Unable to make nested pages

### ISSUE THREAD

**Issue by mortenmoulder** [thecogworks/meganav#11]
Maybe this isn't a feature at all, but it looks like it is.

Your example ([which can be found here](https://our.umbraco.org/media/wiki/269853/636240083201023051_nav-itemsjpg.jpg)) can be translated into a dropdown instead. It would go something like this:

    Home
    Explore ->
        Extend
        Contact
    Blog ->
        Categories

(bear with me, no nested dropdowns)

Let's imagine for a second, that `Explore` and `Blog` are just text/placeholders for the dropdown, and the other 3 (`Extend`, `Contact`, and `Categories`) are children of the dropdowns, is something like that supported with this plugin?

I can't seem to make any nested pages what so ever. I can only make 1 layer of pages. I did set the maximum depth high, so that's definitely not the issue.

To sum it up: I want what this can offer, but also nested pages without a parent page (if that makes sense). Basically just a label/placeholder for some text (think dropdown).

**Comment by SudoCat** [comment]
From what I've experienced using this plugin, if you have your max depth set to at least 2 or higher, then you should be able to nest your items. Simply add a new item, and drag it over to the right hand side until you the dotted box denoting it's position moves about 20px to the right, like so:

![image](https://cloud.githubusercontent.com/assets/5414371/25697835/6d4dbd0a-30b4-11e7-9c9d-c310bca2cab1.png)

Unfortunately, the contrast is a little low, so it's a bit hard to see - but you do need to move the boxes quite far to the right to make them nest.

**Comment by mortenmoulder** [comment]
@SudoCat That looks great, but I was unable to reproduce this. What happens if you try to nest an item that is **NOT** a child? So for example, I see you have **Contact** as a Document Type. What happens if you try to nest a document type such as **News** (assuming you have a news document type)? That's what I'm trying to achieve.

**Comment by mzajkowski** [comment]
@mortenmoulder You can add and nest whatever you want and it not depends on level of the item in the content tree or it's document type. You are a creator and you decide which items you want to add and what levels they should take.

I don't know if this is your case, but probably you want to list / nest all the children of specific node. You can do it programmatically, when you'll be looping through the items in Meganav (to not be forced to keep dynamic list of the items in the navigation object).

Am I missing something?

**Comment by wmhameed** [comment]
Same here I can't make nested items on Umbraco v7.6.3. Dragging item on another item will do nothing except reordering it. The max depth set to 2 and higher.

**Comment by callumbwhyte** [comment]
@mortenmoulder Did you get all of your Meganav questions answered? Let me know if you're still having issues!

**Comment by callumbwhyte** [comment]
@wmhameed @mortenmoulder  I have had a quick look into the symptoms you are describing but cannot seem to replicate the issue. Granted, when setting the "Max Depth" to 2 the navigation is quite restrictive and I can see why the editor experience would be frustrating to use.

Do you have any accurate and detailed steps you can provide to replicate the issue?

I would also highly recommend upgrading to the latest version of Meganav, v1.1.0, which is available via all of the regular channels such as NuGet and Our Umbraco right now!

Cheers :-)

**Comment by jnprovence1** [comment]
For anyone having this same issue, I believe all you have to do is drag the item you want "nested" to the right a little bit.
